### ISSUE TITLE
Own Layout Section Problem in Page Layout

### ISSUE THREAD

**Issue by codiso** [contao/core#8722]
I am using own layout sections in my page Layout. I had defined 8 and wanted to add one more. After I did the 9th section and sved the page the custom sections where empty.
Used Contao Version = 4.3.10

**Comment by codiso** [comment]
I upgraded to 4.3.11 and the problem is still alive.

<img width="805" alt="themes_ spangler_ _seitenlayouts_ _layout_id_1_bearbeiten_-_contao_open_source_cms" src="https://cloud.githubusercontent.com/assets/2002902/26739862/c09a0aaa-47d3-11e7-97f8-c03b69a0fe1f.png">

<img width="774" alt="themes_ spangler_ _seitenlayouts_ _layout_id_1_bearbeiten_-_contao_open_source_cms" src="https://cloud.githubusercontent.com/assets/2002902/26740012/3ef31928-47d4-11e7-8945-7c988803edf8.png">

**Comment by codiso** [comment]
I hope this will be fixed soon …
I am running my first Contao 4 project and this will be a real show stopper :-(

**Comment by fritzmg** [comment]
I am guessing this is because the database field is only a `varchar(1022)`. https://github.com/contao/core-bundle/blob/4.3.11/src/Resources/contao/dca/tl_layout.php#L210

You can try the following, temporary fix: add the file `app/Resources/contao/dca/tl_layout.php` and insert the following content:
```php
$GLOBALS['TL_DCA']['tl_layout']['fields']['sections']['sql'] = 'blob NULL';
```
Then execute `cache:clear` and open the install tool to update the database and try again.

**Comment by codiso** [comment]
I added the file

<img width="197" alt="bildschirmfoto 2017-06-02 um 20 59 04" src="https://cloud.githubusercontent.com/assets/2002902/26740663/bdb2719e-47d6-11e7-8b1e-cbf13eb33ffd.png">

After running the install tool i updated the DB and got that error

<img width="782" alt="bildschirmfoto 2017-06-02 um 21 00 51" src="https://cloud.githubusercontent.com/assets/2002902/26740691/d97e415a-47d6-11e7-8c8a-7276d1c07735.png">

**Comment by codiso** [comment]
Strange, but it works in the backend …
but I can not run the install tool again. After loghing in the same error as before appears :-(

**Comment by fritzmg** [comment]
Ha, funny coincidence ;). Contao 4 uses the field type of the `tl_layout.section` field in order to determine, whether you are on Contao `<=3.2` or not … see [contao/installation-bundle/blob/4.3.11/src/InstallTool.php#L178-L197](https://github.com/contao/installation-bundle/blob/4.3.11/src/InstallTool.php#L178-L197).

This must be fixed in the core (and installation-bundle) of course, but in the mean time you could change [line #196](https://github.com/contao/installation-bundle/blob/4.3.11/src/InstallTool.php#L196) to
```php
return 'blob NULL' !== $column['Type'];
```

**Comment by codiso** [comment]
Do you think this will be fixed in Version 4.4 ?

**Comment by leofeyer** [comment]
See contao/core-bundle#862.
